### ISSUE TITLE
calendarSlideBox have hard-coded time format

### ISSUE THREAD

**Issue by Kehet** [mattlewis92/angular-bootstrap-calendar#93]
In slide box template there are some date formatting hard-coded in library and this is causing headache when I'm trying to localize my calendar ..

[src/templates/calendarSlideBox.html](https://github.com/mattlewis92/angular-bootstrap-calendar/blob/master/src/templates/calendarSlideBox.html#L17) rows 17 -18:
```html
<span ng-show="isMonthView">(<span ng-bind="event.startsAt | date:'shortTime'"></span>)</span>
<span ng-show="isYearView">(<span ng-bind="event.startsAt | date:'MMM d, h:mm a'"></span>)</span>
```

Could these be moved to config or am I missing some already existing config for those?

**Comment by mattlewis92** [comment]
Oh yeah, my bad. I'll fix that and have it deployed in the next hour or so :)

**Comment by Kehet** [comment]
Thanks, that was fast! :+1:

**Comment by mattlewis92** [comment]
Fixed in 0.11.2 :)
